CHANGELOG.md

```diff
@@ -19,7 +21,6 @@
 
 * Image: Don't show `alt` text when loading the image in FireFox. (#80)(#84)
 * Tabs: Update the background color to be transparent for unselected tabs (#79)
-* Heading / Text / SegmentedControl: Add `title` when `truncate` is set (#82)
```

Comment:
@christianvuerings I think this was in the wrong place, it didn't make it out in my release

Reply:
@pfarejowicz Indeed, I'll fix this as part of #93 (new release with only this change)